data/net/minecraft/world/entity/Entity.mapping

```diff
@@ -139,13 +152,21 @@ CLASS net/minecraft/world/entity/Entity
 		ARG 1 livingEntity
 	METHOD getEncodeId ()Ljava/lang/String;
 		COMMENT Returns the string that identifies this Entity's class
+	METHOD getExitPortal (Lnet/minecraft/server/level/ServerLevel;Lnet/minecraft/core/BlockPos;Z)Ljava/util/Optional;
+		ARG 1 world
+		ARG 2 position
+		ARG 3 isNether
 	METHOD getEyeHeight (Lnet/minecraft/world/entity/Pose;)F
 		ARG 1 pose
 	METHOD getEyeHeight (Lnet/minecraft/world/entity/Pose;Lnet/minecraft/world/entity/EntityDimensions;)F
 		ARG 1 pose
 		ARG 2 size
 	METHOD getEyePosition (F)Lnet/minecraft/world/phys/Vec3;
 		ARG 1 partialTicks
+	METHOD getFluidHeight (Lnet/minecraft/tags/Tag;)D
+		ARG 1 fluidTag
+	METHOD getHorizontalDistanceSqr (Lnet/minecraft/world/phys/Vec3;)D
+		ARG 0 vec3
```

Comment:
Abbreviations should be avoided.

Reply:
I see, then, what should I use for parameter names that cannot be referenced from any other parts of the code?
For example, `Object#equals(Object)` method's parameter names are always `p_equals_1_`. What name for such function would be good?
Also, Is it recommended to change parameter names of lambda functions? If so, is it allowed to use `_unused` when parameters are unused?

Reply:
Either `other` or `object` are fine for the equals method, I believe. Yes, lambda functions should also be mapped. If they are never used and what they are can't be interpreted from uses then `unused` or similar should be fine.

Reply:
Made some changes to mapping following your advice

Reply:
Does it violate any other mapping standard now?